static/css/legacy.less

```diff
@@ -1683,6 +1682,13 @@ ul#siteSearch {
     color: @grey;
   }
 
+  // openlibrary/macros/SearchResultsWork.html
```

Comment:
Sorry I missed this before, but please use the existing rule here rather than duplicate it, since they are the same thing like you've done below.

`span.resultTitle {` above  should become

```
.resultTitle {
```